### DISCUSSION TITLE
Roadmap: Alternate Relationship Field UI

### DISCUSSION THREAD

**Discussion by jmikrut** [payloadcms/payload#285] [Roadmap]
### Description

The current `relationship` field UI is very similar to the `select` field type UI. This field is heavily utilized for many separate applications, and there are opportunities to better equip admins with a relationship-specific field interface. This feature will require a large amount of user experience design, wherein we will be looking to our community for feedback on our proposed approaches. This feature will be presentational only and have no effect on data.

### Implementation Detail

Add a new field config property to all `relationship` field types that will allow developers to specify which type of `relationship` field to be displayed within the admin UI.  

### Potential Breaking Changes

None

### Effort

Medium

**Comment by DanRibbens** [comment]
Related discussions:
https://github.com/payloadcms/payload/discussions/782
https://github.com/payloadcms/payload/discussions/763
